frontend/package.json

```diff
@@ -27,7 +27,10 @@
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
-    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-devconsole": "cd packages/dev-console/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-devconsole-headless": "cd packages/dev-console/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/project-creation.feature\";",
```

Comment:
Will these tests run as part of CI? Do they run if I use `yarn run test-cypress-headless`?

Reply:
Yes, `test-cypress-headless` is now:
` "test-cypress-headless": "yarn run test-cypress-console-headless && yarn run test-cypress-devconsole-headless",`

Reply:
But how will this scale if every package adds a test?

Reply:
@makambalaji i thought you weren't planning to run all the dev-console tests on CI because you said it took hours to run.

Reply:
@spadgett - From dev-console side, Will select the test scenarios for execution based on the tag names mentioned in [cypress.json](https://github.com/openshift/console/blob/bd7902835c50c0959a32e6915265af3617d32a9e/frontend/packages/dev-console/integration-tests/cypress.json)

@christianvogt - In CI, will run only smoke tests, now the execution time is also reduced from 2.5 hr to 1 hr.

Reply:
This pr doesn't add 1hr of tests. I'd like to get the smoke tests down below 1hr down before adding all those tests.

Reply:
> But how will this scale if every package adds a test?

@spadgett I was envisioning teams adding to `test-cypress-headless` in the same manor.  Teams need to `cd` to their cypress dir, so it seems to be the correct method.